.eslintrc

```diff
@@ -58,5 +58,13 @@
     "ecmaFeatures": {
       "legacyDecorators": true
     }
-  }
+  },
+  "overrides": [
+    {
+      "files": ["**.info.js", "**.info.jsx"],
+      "rules": {
+        "react/prop-types": 0
+      }
+    }
+  ]
```

Comment:
This is now showing additional console errors on every single page:
![image](https://user-images.githubusercontent.com/31325167/124752635-64565f00-df28-11eb-8fc7-af06bda45964.png)